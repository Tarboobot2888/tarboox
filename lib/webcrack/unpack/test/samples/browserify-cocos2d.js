window.__require = (function t(e, n, i) {
  function r(a, o) {
    if (!n[a]) {
      if (!e[a]) {
        var l = a.split('/');
        l = l[l.length - 1];
        if (!e[l]) {
          var c = typeof __require == 'function' && __require;
          if (!o && c) {
            return c(l, true);
          }
          if (s) {
            return s(l, true);
          }
          throw new Error("Cannot find module '" + a + "'");
        }
      }
      var u = (n[a] = {
        exports: {},
      });
      e[a][0].call(
        u.exports,
        function (t) {
          return r(e[a][1][t] || t);
        },
        u,
        u.exports,
        t,
        e,
        n,
        i,
      );
    }
    return n[a].exports;
  }
  var s = typeof __require == 'function' && __require;
  for (var a = 0; a < i.length; a++) {
    r(i[a]);
  }
  return r;
})(
  {
    BattleHint: [function (t, e, n) {}, {}],
    GChangeCount: [function (t, e, n) {}, {}],
    'pako_deflate.min': [function (t, e, n) {}, {}],
  },
  {},
  ['BattleHint', 'GChangeCount'],
);
