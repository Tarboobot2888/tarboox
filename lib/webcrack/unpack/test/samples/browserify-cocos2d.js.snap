BrowserifyBundle {
  "entryId": "BattleHint",
  "modules": Map {
    "BattleHint" => BrowserifyModule {
      "ast": ,
      "dependencies": {},
      "id": "BattleHint",
      "isEntry": true,
      "path": "index.js",
    },
    "GChangeCount" => BrowserifyModule {
      "ast": ,
      "dependencies": {},
      "id": "GChangeCount",
      "isEntry": false,
      "path": "./GChangeCount.js",
    },
    "pako_deflate.min" => BrowserifyModule {
      "ast": ,
      "dependencies": {},
      "id": "pako_deflate.min",
      "isEntry": false,
      "path": "./pako_deflate.min.js",
    },
  },
  "type": "browserify",
}